wd_helper.au3

```diff
@@ -1859,8 +1859,8 @@ Func _WD_ElementActionEx($sSession, $sElement, $sCommand, $iXOffset = Default, $
 
 	EndSwitch
 
-	; This line in compilation process will be linearized, and will be processed once, thus next usage will be significantly faster
-	; $sActionTemplate declaration is outside the switch to not pollute simplicity of the >Switch ... EndSwitch< - for better code maintenance
+	; $sActionTemplate declaration is outside the switch to not pollute simplicity of the >Switch ... EndSwitch< - for better code maintaince
+	; This following line in compilation process will be linearized, and will be processed once, thus next usage will be significantly faster
```

Comment:
I don't believe this is needed or helpful to the reader because --

- "This line" implicitly references the next line of code IMO, so the word "following" isn't needed
- "this following" isn't proper English. "The following" is better syntax *if* you don't agree with the above
- My prior commit fixed a spelling error, which you reintroduce here

Reply:
Thanks really for the English lesson. As you know, I am self-taught in this regard.
Will fix it.